src/EnergyPlus/ThermalISO15099Calc.cc

```diff
@@ -116,43 +116,47 @@ void film(Real64 const tex, Real64 const tw, Real64 const ws, int const iwd, Rea
     Real64 bexp;
 
     // calculation of convection component of exterior film coefficient using the :
-    {
-        auto const SELECT_CASE_var(ibc);
-        if (SELECT_CASE_var == 0) { // ISO 15099
-            hcout = 4.0 + 4.0 * ws;
-        } else if (SELECT_CASE_var == -1) { // old ASHRAE SPC142 correlation
-            if (iwd == 0) {                 // windward
-                if (ws > 2.0) {
-                    vc = 0.25 * ws;
-                } else {
-                    vc = 0.5;
-                }
-            } else { // leeward
-                vc = 0.3 + 0.05 * ws;
-            }
-            hcout = 3.28 * std::pow(vc, 0.605);
-            hcout *= conv;                  // convert to metric
-        } else if (SELECT_CASE_var == -2) { // Yazdanian-Klems correlation:
-            if (iwd == 0) {                 // windward
-                acoef = 2.38;
-                bexp = 0.89;
-            } else { // leeward
-                acoef = 2.86;
-                bexp = 0.617;
+    switch (ibc) {
+    case 0: { // ISO 15099
+        hcout = 4.0 + 4.0 * ws;
+    } break;
+    case -1: {          // old ASHRAE SPC142 correlation
```

Comment:
What is the fascination with using negative numbers for enumeration values in EnergyPlus?

Reply:
I think it was just a trick to make it easier to inspect variables during debugging sessions.  Because there was no type safety, they were all just integers and could be used all willy-nilly it was easier to spot when something was incorrectly assigned if it had a value of -6.  Not saying it's right, that's just what I remember from debugging Fortran in CVF.

Reply:
At least you used integers and not floats.